sputnikdao2/src/views.rs

```diff
@@ -0,0 +1,100 @@
+use std::cmp::min;
+
+use crate::*;
+
+/// This is format of output via JSON for the proposal.
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+pub struct ProposalOutput {
+    /// Id of the proposal.
+    pub id: u64,
+    #[serde(flatten)]
+    pub proposal: Proposal,
+}
+
+/// This is format of output via JSON for the bounty.
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+pub struct BountyOutput {
+    /// Id of the bounty.
+    pub id: u64,
+    #[serde(flatten)]
+    pub bounty: Bounty,
+}
+
+#[near_bindgen]
+impl Contract {
+    /// Returns semver of this contract.
+    pub fn version(&self) -> String {
+        env!("CARGO_PKG_VERSION").to_string()
+    }
+
+    /// Returns config of this contract.
+    pub fn get_config(&self) -> Config {
+        self.data().config.clone()
```

Comment:
with LazyOption, it is 
```suggestion
        self.data().config.get().unwrap()
```